spec/lib/fee_threshold_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe FeeThreshold do
+  let(:application) { create :application }
+  subject { described_class.new(application) }
+
+  context '.band' do
```

Comment:
This is a really really minor thing, but `.` is by convention used for class methods and `#` is used for instance methods.